config/packages/reset_password.yaml

```diff
@@ -1,2 +1,2 @@
 symfonycasts_reset_password:
-  request_password_repository: App\Repository\PasswordResetRequestRepository
\ No newline at end of file
+  request_password_repository: App\Repository\ResetPasswordRequestRepository
```

Comment:
indentation fixed in [flex recipe](https://github.com/jrushlow/reset-password-bundle-flex/blob/master/1.0/config/packages/reset_password.yaml)

Reply:
In the recipe, a tweak to the comment on top:

```
    # Replace symfonycasts.reset_password.fake_request_repository with the full
    # namespace of the password reset request repository after it has been created.
    # i.e. App\Repository\ResetPasswordRequestRepository
```

Reply:
done